bubblesub/ui/views.py

```diff
@@ -57,36 +60,60 @@ def __str__(self) -> str:
     ConsoleInput = "console-input"
 
 
-class ViewLayoutManager(QtCore.QObject):
+VIEW_WIDGET_VISIBILITY_MAP = {
+    View.Full: {
+        TargetWidget.Spectrogram: True,
+        TargetWidget.VideoContainer: True,
+        TargetWidget.StatusAudioLabel: True,
+        TargetWidget.StatusFrameLabel: True,
+    },
+    View.Audio: {
+        TargetWidget.Spectrogram: True,
+        TargetWidget.VideoContainer: False,
+        TargetWidget.StatusAudioLabel: True,
+        TargetWidget.StatusFrameLabel: False,
+    },
+    View.Video: {
+        TargetWidget.Spectrogram: False,
+        TargetWidget.VideoContainer: True,
+        TargetWidget.StatusAudioLabel: False,
+        TargetWidget.StatusFrameLabel: True,
```

Comment:
Disable the volume handler for the video view, please

Reply:
But why? You ought to be able to control volume of the played video

Reply:
Sure, but I was going to remove that because I wanted to leave only the widgets necessary to work with videos. You can always control volume through shortcuts in video view.

Reply:
Which shortcuts?

>Disable the volume handler for the video view,

>You can always control the volume through shortcuts in video view.

Aren't these two statements contradictory to each other?

Reply:
Uh, don't we have shortcuts to control volume? That's pretty bad, I didn't know that. 

Wrong verb, disable = hide in that case.

Reply:
OK by shortcuts you meant keyboard shortcuts. No, we don't.

Reply:
Feel free to add whatever you prefer, it's the same for me.

So the plan will be:
- Hide the volume handler in video view
- Use keyboard shortcuts to turn up/down the volume in video view

Reply:
TBH I'd leave the volume slider and hide it on your side in the config, or by issuing the command `show-widget video-volume -m hide`.

Reason - we're going to improve its placement anyway. Possibly place it in the statusbar.

Another option is to have "toggle volume slider" in the menu bar similar to how we have "hide video controls".

The volume shortcuts are an orthogonal issue.

Reply:
> Reason - we're going to improve its placement anyway. Possibly place it in the statusbar.

If I could, I would completely remove it in favour of keyboard shortcuts only. More space on the scree, but this one is another issue, we don't need to discuss it here
> 
> Another option is to have "toggle volume slider" in the menu bar similar to how we have "hide video controls". 

Let's add this one in this PR for now, please.

Reply:
Done, LMK if it's good to go

Reply:
Yep, you can land it, thanks a lot! 😁